src/sql/strong-oracle/drop.sql

```diff
@@ -1,4 +1,14 @@
 BEGIN
+   EXECUTE IMMEDIATE 'DROP TABLE "L_TRANSFER_ADJUSTMENTS"';
+EXCEPTION
+   WHEN OTHERS THEN
+      IF SQLCODE != -942 THEN
+         RAISE;
+      END IF;
+END;
+/
+
```

Comment:
Don't you need to also drop the indexes and triggers, or is that going to be another PR?